backend/node-js-express/node-js-express-postgresql/app/app.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express');
+
+// Load environment variables from .env file
+require('dotenv').config();
+
+const RootController = require('./root/root.controller');
+const HealthController = require('./health/health.controller');
+const UsersController = require('./user/user.controller');
+
+class App {
+  static bootstrap() {
+    return new App();
+  }
+
+  constructor() {
+    // create express js application
+    this.app = express();
+
+    // Middleware for parsing JSON bodies
+    this.app.use(express.json());
+
+    // add routes
+    this.routes();
+  }
+
+  routes() {
+    this.app.use('/', RootController);
+    this.app.use('/health', HealthController);
```

Comment:
This endpoint should be `healthz` as per the health checks path in helm templates.

Reply:
Ah ok, makes sense, thanks!